Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -1962,7 +1962,23 @@
     <!-- XACT: Button/Dialog label to cancel something-->
     <string name="generic_action_abort">"Cancel"</string>
     <!-- XACT: Button/Dialog label to remove something-->
-    <string name="generic_action_remove">"Remove"</string>
+    <string name="generic_action_remove">Remove</string>
```

Comment:
@chiljamgossow we need german strings, english strings in this file won't be recognized by the translation system. Please sync with @janetback for the correct strings.